packages/babel-parser/src/util/whitespace.ts

```diff
@@ -22,7 +22,7 @@ export function isNewLine(code: number): boolean {
 export const skipWhiteSpace = /(?:\s|\/\/.*|\/\*[^]*?\*\/)*/g;
 
 export const skipWhiteSpaceInLine =
-  /(?:[^\S\n\r\u2028\u2029]|\/\/.*|\/\*.*?\*\/)*/y;
+  /(?:[^\S\n\r\u2028\u2029]|\/\/.*|\/\*.*?\*\/)*/g;
```

Comment:
This change has no effect since we are not using `.exec()`, here I am just aligning to the flag of `skipWhiteSpace`.